### PR TITLE
Gcp seft message

### PR DESCRIPTION
**Pull request by insacuri**
# What and why?

# How to test?

# Trello

### CODE REVIEW

File: application/controllers/gcp_survey_response.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+    """
+    The survey response from a respondent
+    """
+    def add_survey_response(self, case_id: str, file, file_name: str, survey_ref: str):
+        """
+        Encrypt and upload survey response to gcp bucket, and put metadata about it in pubsub.
+
+        :param case_id: A case id
+        :param file: A file object from which we can read the file contents
```
Comment (warrenbailey) [inline application/controllers/gcp_survey_response.py:49]: Isn't this actually the file contents now?

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: application/controllers/survey_response.py [hunk 2]
```diff
@@ -42,19 +43,18 @@ def add_survey_response(self, case_id, file, file_name, survey_ref):
         """
 
         tx_id = str(uuid.uuid4())
-        log.info('Adding survey response file', filename=file_name, case_id=case_id, survey_id=survey_ref, tx_id=tx_id)
-        file_contents = file.read()
-        file_size = len(file_contents)
+        bound_log = log.bind(filename=file_name, case_id=case_id, survey_id=survey_ref, tx_id=tx_id)
+        bound_log.info('Adding survey response file')
+        file_size = len(file)
```
[the remark this answers is not in the document]
Comment (insacuri) [reply application/controllers/survey_response.py]: Done

File: application/controllers/gcp_survey_response.py (hunk 1 above)
Comment (insacuri) [reply application/controllers/gcp_survey_response.py]: Done

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**